scripts/inject-probe-data.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import argparse
+import contextlib
+import json
+import sys
+
+
+def parse_cmdline() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
Would you add a short help string explaining the problem being solved?

Reply:
Sure thing!